README.md

```diff
@@ -132,6 +132,50 @@ until the constructor for `config.Provider` can be fully satisfied.
 If resolution is not possible (for instance, if one of the required dependencies
 lacks a constructor and doesn't appear in the graph), `dig` returns an error.
 
+## Invoke
+
+The `Invoke` API executes the provided function and stores returned results in the dig container.
+
+`Invoke` looks through the dig graph and resolves all the constructor parameters for execution.
+At the end of method invokation, `Invoke` inserts return results except the `error` object, into the
```

Comment:
Yeah, how about "Once the method is invoked, `Invoke` inserts all return(ed?) results except the `error` object into the…"? It's a little weird because we're saying the `Invoke` method does something after the method is invoked. But I think it's OK. What do you think?